.env

```diff
@@ -1,4 +1,5 @@
 NODE_ENV=development
```

Comment:
Hello @willyserge It's not good to push the `environment variables`. 
you need to add `.env` to the `.gitignore`

Reply:
done